taichi/backends/vulkan/spirv.hpp

```diff
@@ -0,0 +1,2203 @@
+// Copyright (c) 2014-2020 The Khronos Group Inc.
```

Comment:
Let's remove this header and ask Vulkan devs to install it instead... https://github.com/KhronosGroup/SPIRV-Headers#how-to-install-the-headers